scripts/MRK17_Exp1_rk01.jmd

```diff
@@ -103,7 +104,7 @@ Sometimes RE structures are more conviently specified with indicator variables (
 @ level of contrasts) than the factors.
 
 ```julia
-mm = Int.(cmplxLMM.X);
+mm = Int.(cmplxLMM.X);  # Please don't do this.
```

Comment:
As I said, I really want us to avoid getting the numeric model matrix and extracting columns from it.  This is a **very bad and dangerous practice**